lq1/default.cfg

```diff
@@ -58,13 +59,44 @@ bind	"-"		"sizedown"
 +speed
 
 //Cvars
-gl_texturemode "GL_NEAREST_MIPMAP_LINEAR"
-gl_texture_anisotropy "8"
-registered "1""
-vid_fsaamode "1"
-vid_fsaa "4"
-volume "0.7""
-
+gl_texturemode gl_nearest_mipmap_linear
+gl_texturemode_sky gl_nearest
+gl_texturemode_hud gl_nearest
+gl_texturemode2d gl_nearest
+gl_overbright 1
+gl_overbright_model 1
+gl_smoothfont 0
+gl_texture_anisotropy 16
+vid_width "1280"
+vid_height "800

```

Comment:
See comment in `-config.cfg` as to why forcing a default high resolution is problematic.

Reply:
sadly does break on lower then 800p screens
the player would have to go change the res them selfs
will remove -config.cfg since its no longer needed